sdk/metricsadvisor/ai-metrics-advisor/test/public/keyRotation.spec.ts

```diff
@@ -14,13 +14,13 @@ describe("MetricsAdvisorKeyCredential", () => {
 
   it("update subscriptionKey", async function() {
     const newSubscriptionKey = "Abc";
-    credential.updateSubscriptionKey(newSubscriptionKey);
+    credential.updateKey({ subscriptionKey: newSubscriptionKey });
```

Comment:
could you please rename `newSubscriptionKey` to `newValue`? I suspect that subscriptionkey is triggering credscan warning.

Reply:
done